### PR TITLE
Updated Community Section in Readme

### LINKED ISSUE #211: [README] Update README Community Section
**Description**
The Community section in the README says "Access the Community Drive". There is a new procedure of having this automated based on the individual filling in a [community member form](https://layer5.io/newcomer)

**Expected Behavior**
Update "Access Community Drive" to "To Access Community Drive, fill Community Member Form"

**Screenshots**
img

**Enviroment:**
 - OS: [e.g. Ubuntu]
 - Browser: [e.g. Chrome, Safari]
 - Version: [e.g. 22]
 - Device: [e.g. laptop, iPhone 8]

### PR DESCRIPTION
**Pull request by DARK-art108**
**Description**

Updated Community Section in Readme

This PR fixes #211 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by DARK-art108** [comment]
Hello @leecalcote ,take a look at the PR and consider the commit : `21e19e2c37a50b9cd4408663f69749c2f1a6f415`

**Comment by DARK-art108** [comment]
@leecalcote Please take a look at the PR :)

**Comment by leecalcote** [comment]
Thank you, @DARK-art108

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Our projects are community-built and welcome collaboration. 👍 Be sure to see
 <p>
 ✔️ <em><strong>Join</strong></em> any or all of the weekly meetings on the <a href="https://calendar.google.com/calendar/b/1?cid=bGF5ZXI1LmlvX2VoMmFhOWRwZjFnNDBlbHZvYzc2MmpucGhzQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20">community calendar</a>.<br />
 ✔️ <em><strong>Watch</strong></em> community <a href="https://www.youtube.com/channel/UCFL1af7_wdnhHXL1InzaMvA?sub_confirmation=1">meeting recordings</a>.<br />
-✔️ <em><strong>Access</strong></em> the <a href="https://drive.google.com/drive/u/4/folders/0ABH8aabN4WAKUk9PVA">community drive</a>.<br />
+✔️ <em><strong>To Access the Community Drive</strong></em>, fill <a href="https://layer5.io/newcomer">Community Member Form</a>.<br />
```
Comment (leecalcote) [inline README.md:35]: lowercase "access"

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (DARK-art108) [reply README.md]: Done @leecalcote Added Suggested Changes!

**Review by leecalcote — APPROVED** [review]
Thanks, @DARK-art108 👍

**Pull request merged**